packages/flutter_tts/pubspec.yaml

```
@@ -2,7 +2,7 @@ name: flutter_tts_tizen
 description: The Tizen implementation of flutter_tts plugin.
 homepage: https://github.com/flutter-tizen/plugins
 repository: https://github.com/flutter-tizen/plugins/tree/master/packages/flutter_tts
-version: 1.3.0
+version: 1.3.1
```

Minor version update is expected when adding new features?
```suggestion
version: 1.4.0
```

---

done